charts/reform-scan-notification-service/values.yaml

```diff
@@ -22,6 +22,8 @@ java:
     FLYWAY_SKIP_MIGRATIONS: true
     QUEUE_NOTIFICATIONS_MAX_RETRY: 5
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    NOTIFICATIONS_CONSUME_TASK_DELAY: 3000
```

Comment:
do we want same value for AAT/Preview and  PROD

Reply:
we can not run functional test. so it can be same?

Reply:
Worth having a comment about what units of time we're using

Reply:
> we can not run functional test. so it can be same?

I think we can run functional tests any reason why not ?

publish message to queue.
Consumer consumes the messages and saves in DB
Retrieve notification through endpoint and validate ?

Reply:
> Worth having a comment about what units of time we're using

so better to change name because it may overrriden in different places
how about NOTIFICATIONS_CONSUME_TASK_INTERVAL_IN_MS ?